### ISSUE TITLE
How do you verify the checksum if you created it with -includelastdir?

### ISSUE THREAD

**Issue by a84r7a3rga76fg** [idrassi/DirHash#36]
I want to verify `document.txt` in `C:\test\[directory]`.

```
PS C:\test> cat checksum.blake3; ls -recurse; dirhash "C:\test" Blake3 -verify "C:\test\checksum.blake3" -nologo -nowait
AF1349B9F5F9A1A6A0404DEA36DCC9499BCB25C9ADC112B7CC9A93CAE41F3262  [directory]\document.txt

    Directory: C:\test

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          2024-10-30    14:43                [directory]
d----          2024-10-30    14:52                [second directory]
-a---          2024-10-30    14:45             95 checksum.blake3

    Directory: C:\test\[directory]

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          2024-10-30    14:42              0 document.txt

    Directory: C:\test\[second directory]

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          2024-10-30    14:51              0 picture.txt
Using Blake3 to verify hash of "C:\test" ...
Error: file "C:\test\[second directory]\picture.txt" not found in checksum file.
```

**Comment by idrassi** [comment]
The error you have is about a file in target directory `C:\test\`  **not found in checksum file**. This is considered an error since it means content of the folder changed (new file was added).

If you want to only verify the content of `[directory]`, then you should specify it as a target and add `-includeLastDir` switch to the command: 
`dirhash "C:\test\[directory]" Blake3 -verify "C:\test\checksum.blake3" -includeLastDir -nologo -nowait`

Basically, when you specify  `-includeLastDir` during SUM computation, you also need to specify  `-includeLastDir` during SUM verification.

I hope this clarifies things.
